### DISCUSSION TITLE
algebra relazionale - 8

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#65] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/3f240536-fe24-457f-97c4-9d15871fb365)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/f258e0bc-8aa2-4021-ad2c-ac661529c13a)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-16 165504](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/84105857-10b7-46ed-a690-c8148b943f18)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/1899a9f9-cc87-4696-b417-ec2e60766bdc)
